assets/img/sponsors/TMG.svg

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
```

Comment:
You don't need a lot of this text. Take a look at existing SVGs and remove what they don't have. (The DOCTYPE, a lot of properties on the `<svg>`, metadata, etc.)